drf_client/connection.py

```diff
@@ -311,6 +312,14 @@ def login(self, password, username=None):
 
         payload = json.dumps(data)
         r = requests.post(url, data=payload, headers=DEFAULT_HEADERS)
+
+        if r.status_code == 401:
```

Comment:
I don't think we can just assume that if we get a 401, it means MFA.
Can you check what `r.content` shows? I hope we output something we can do to allow this to be something like
```
if status == 401 and content == "MFARequired":
   ....
```